ArduCopter/mode_guided.cpp

```diff
@@ -516,7 +516,7 @@ void ModeGuided::posvel_control_run()
     // process pilot's yaw input
     float target_yaw_rate = 0;
 
-    if (!copter.failsafe.radio && ((copter.g2.auto_options & (uint32_t)Options::IgnorePilotYaw) == 0)) {
+    if (!copter.failsafe.radio && ((copter.g2.guided_options & (uint32_t)Options::IgnorePilotYaw) == 0)) {
```

Comment:
nice one ! better use use_pilot_yaw() like on other functions !

Reply:
Thank you for the review!
I fixed it, and tested it in SITL again.